streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaStreamingContext.scala

```diff
@@ -396,7 +396,7 @@ class JavaStreamingContext(val ssc: StreamingContext) extends Closeable {
    * Create an input stream from a queue of RDDs. In each batch,
    * it will process either one or all of the RDDs returned by the queue.
    *
-   * NOTE:
+   * @note
```

Comment:
Yup, it seems keeping the original format at least.

**Before**

- Scala
  <img width="895" alt="2016-11-18 1 45 48" src="https://cloud.githubusercontent.com/assets/6477701/20419048/c067da80-ad96-11e6-84e1-6f54e6e559a7.png">

- Java
  <img width="707" alt="2016-11-18 1 46 07" src="https://cloud.githubusercontent.com/assets/6477701/20419049/c067f18c-ad96-11e6-9653-947e49f8b978.png">


**After**

- Scala
  <img width="694" alt="2016-11-18 1 44 58" src="https://cloud.githubusercontent.com/assets/6477701/20419066/e5aefada-ad96-11e6-860c-d298f96bd18b.png">

- Java
  <img width="764" alt="2016-11-18 1 45 26" src="https://cloud.githubusercontent.com/assets/6477701/20419065/e5adbc88-ad96-11e6-8139-72059b3c7341.png">